public/js/charts.js

```diff
@@ -0,0 +1,148 @@
+function getDataForDoughnutChart () {
+
+    var ctx = document.getElementById("doughnutChart").getContext('2d'),
+        data = {
+            datasets: [{
+                data: [10, 20, 30],
+                backgroundColor: [
+                    '#ea04ad',
+                    '#36a2eb',
+                    '#cc65fe'
+                ]
+            }],
+            labels: [
+                'Rozrywka',
+                'Yellow',
+                'Blue'
+            ]
+        }
+    
+
+    var myDoughnutChart = new Chart(ctx, {
+        type: 'doughnut',
+        data: data
+    });
+}
+function getDataForDoughnutCharts () {
+    
+        var ctx = document.getElementById("doughnutCharts").getContext('2d'),
```

Comment:
śmieszne... raz IDik doughnutChart a tu doughnutCharts, weź nazwij to normalnie, zgodnie z przeznaczeniem